tests/common.py

```diff
@@ -61,6 +61,7 @@ def vllm_setup_test(runner, args, mock_popen, *_mock_args):
 
     return mock_popen.call_args_list[0][1]["args"]
 
+
```

Comment:
Do we need these newlines?

Reply:
updated in a future commit. I can go back if you'd like and remove them

Reply:
I see that now! No worries, I'll resolve this convo